### ISSUE TITLE
Add to List popup is not scrollable

### ISSUE THREAD

**Issue by tophercully** [tophercully/snippp#59]
When adding a snippet to a list on the browser or dashboard, the popup modal does not contain the overflow, instead it can even overflow the page without becoming scrollable.

**Comment by charliepaull** [comment]
hi @tophercully, I'd love to work on this issue. Do you have any initial guidance that can help me get started? I am still new to open-source contributions.

**Comment by tophercully** [comment]
Howdy @charliepaull! This is a simple one I listed to test the request approval flow so perfect! 

A: It's all styled in Tailwind

B: Inside `src/components/browserComponents/Display.tsx:724` there's a div that renders all userLists into it. It has a `max-height` set but the content overflows that max and needs to be styled to scroll. Preferably keep the Add List button outside of this scrollable div so it's always visible!

I'll be adding issues to this project throughout the week because this Display component is far too long and can be broken up, which will also be some pretty simple issues to work on. I'll be sure to add more details to future issue posts as well :)

**Comment by tophercully** [comment]
I'm also still working on a dummy-data solution or temporary dev key for contributors. So if you do need dummy data to test with a full list of Lists feel free to create your own and reconnect what's needed before you submit it

**Comment by tophercully** [comment]
I've just finished a project rebuilding the app in Nextjs for SEO and indexing purposes, and tackled this issue while working on that! I've also migrated the database to Neon so we have a branchable system and I can bring people in with branches of the DB without posing a danger to the production DB, instead of relying on dummy data like I suggested originally.

Closing this now but I'll have more up soon with a simpler contribution process, sorry about how unmanageable this first issue was!
